Assets/MixedRealityToolkit.Examples/Demos/StandardShader/New feature/MaterialParameterCecker.cs

```diff
@@ -0,0 +1,374 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.UI;
+using Microsoft.MixedReality.Toolkit.UI;
+
+
+public class MaterialParameterCecker : MonoBehaviour
+{
+    //TargetMaterial
```

Comment:
add /// comment documentation to all public fields and methods for API documentation